code/socialDistribution/static/js/comment/comment.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Redirect to home page of author
+ * */
+function redirectToAuthorPage(id) {
+    location.href = `${location.protocol}//${location.host}/app/author/${id}`;
+}
```

Comment:
I think this works great, but consider that we could probably wrap the elements in a href tag, which might be a bit simpler.

Reply:
I think we should use onclick event handlers instead of wrapping elements with an anchor tag, as the anchor tag has its own default styles.

Reply:
This just seems a bit unconventional to me. I've never seen this before. Navigating between web pages is the whole purpose of the anchor tag, and default styles can be overwritten. Regardless, we can keep it like this. Thanks